pipeline/scraper/pypi_query.py

```diff
@@ -56,12 +57,30 @@ def get_package_metadata(dependency):
     entry['name'] = dependency
 
     result = json.loads(pypistats.recent(dependency_name, "month", format="json"))
-    print(result)
-    entry['downloads_last_month'] = result['data']['last_month']
-    request_url = f'{PYPI_DEPENDENCY_META_URL}{dependency_name}/{dependency[version_symbol_index+1:]}/json'
-    json_result = requests.get(request_url)
-    print(request_url)
-    print(json_result)
-    return entry
 
-get_package_metadata('pkg:pypi/gunicorn@0.1')
\ No newline at end of file
+    entry['monthly_downloads_last_month'] = result['data']['last_month']
+    entry['monthly_downloads_a_year_ago'] = 0
+
+    try:
+        request_url = f'{PYPI_DEPENDENCY_META_URL}{dependency_name}/json'
+        json_result = requests.get(request_url).json()
+    except:
```

Comment:
Would be nice to print the exception message for debugging purposes.

Reply:
Also, bare exception is not optimal. pylint or even pycodestyle will be screaming at this line